### ISSUE TITLE
Nightly build completely broke controller support

### ISSUE THREAD

**Issue by lyndonguitar** [LizardByte/Sunshine#1407]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

automated nightly release - 2023-06-28T02:12:10Z - https://github.com/LizardByte/Sunshine/commit/12e67740351fbeec495455e1509c9298a924720c

completely broke controller support, controllers are not calibrated anymore. reverting to latest stable version fixes it

### Expected Behavior

Controller should work and should be properly calibrated

### Additional Context

joysticks are wonky, buttons are wrongly mapped, not work as intended

### Host Operating System

Windows

### Operating System Version

Windows

### Architecture

64 bit

### Sunshine commit or version

automated nightly release - 2023-06-28T02:12:10Z - 12e6774

### Package

Windows - installer

### GPU Type

Nvidia

### GPU Model

RTX 3080

### GPU Driver/Mesa Version

535.98

### Config

```Shell
its empty
```

### Relevant log output

```shell
was not able to get it when it was not working
```

**Comment by cgutman** [comment]
We'll need more information about your setup.

What client version are you using? What controller is connected to your client? What controller do you have Sunshine set to emulate?

Which buttons and axes are not mapped correctly? What are those incorrectly mapped buttons/axes actually mapped to?

**Comment by Lucide** [comment]
* Windows 10 host
* "xbox series x" controller (wireless/wired), **classic xbox 360 wired controller**
* android, linux, windows clients

Button mapping is scrambled at least from build 11aedf56a24cbf1e82f9699a39e4207346aefa5a.

The vertical left axis acts as the right horizontal axis, and the left horizontal acts as the left vertical axis. Basically the same as [moonlight's !822](https://github.com/moonlight-stream/moonlight-qt/issues/822). The d-pad and face buttons are ok.

This is different from the scrambled mappings of cabled-vs-bluetooth controllers found with certain games/situations (where the stick axis are fine but the shoulder buttons/axis and the d-pad are incorrect).

**Comment by cgutman** [comment]
Please test this build from #1426 which should fix the issue: https://github.com/LizardByte/Sunshine/suites/14124093845/artifacts/790595658

**Comment by Lucide** [comment]
Fixed in my use cases🙂

**Comment by erictbar** [comment]
I had the same issue and this test build fixed it for me

Host: Windows 11 Pro 22H2 22621.1928, Latest nightly
Clients: 
- Nintendo Switch running Android 11, wired connected JoyCons
- Xbox Series S running Moonlight for Xbox UWP, Xbox USB-C controller over Bluetooth

The game I noticed the issue with was Mass Effect Legendary Edition. Moving around with the L and R sticks was completely broken. When I went to controller settings in Windows and went to calibrate, I was able to get the Left controller to the upper left, but would not find the lower left.
